docs/api/cases/cases-api-find-connectors.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[cases-api-find-connectors]]
+== Find connectors API
+++++
+<titleabbrev>Find connectors</titleabbrev>
+++++
+
+Retrieves a paginated subset of <<action-types,connectors>>.
+
+In particular, only the connectors that are supported for use in cases are
+returned. Refer to the list of supported external incident management systems in
+<<add-case-connectors>>.
+
+=== Request
+
+`GET <kibana host>:<port>/api/cases/configure/connectors/_find`
+
+=== Prerequisite
+
+You must have `read` privileges for the *Actions and Connectors* feature in the *Management* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>.
+
+=== Response code
+
+`200`::
+   Indicates a successful call.
+
+=== Example
+
+[source,sh]
+--------------------------------------------------
+GET api/cases/configure/connectors/_find
+--------------------------------------------------
+// KIBANA
+
+The API returns a JSON object describing the connectors and their settings:
+
+[source,json]
+--------------------------------------------------
+[{
```

Comment:
This example output for the find connectors API has changed a lot in my tests from what was shown in https://www.elastic.co/guide/en/security/master/cases-api-find-connectors.html. Given that there are no longer "pages", should we omit "paginated" from the introductory sentence?

Reply:
You are right! The response in the docs is wrong. I do not recall this kind of response tbh. And yes, we should remove the paginated is not correct. See my previous comment 🙂.

@nastasha-solomon I think it should also be fixed for the Security solution until we take ownership of the Cases API.

Reply:
Thanks for finding this @lcawl and for the recommendation @cnasikas! Just so I have the correct information, would you mind providing an example response that users can expect to see? Here's the response I got through my testing:

````
[
    {
        "closure_type": "close-by-user",
        "owner": "securitySolution",
        "created_at": "2022-03-17T21:03:50.648Z",
        "created_by": {
            "username": "docsuser",
            "email": "",
            "full_name": ""
        },
        "updated_at": null,
        "updated_by": null,
        "connector": {
            "id": "3331d660-98a6-11ec-8ee2-c385e0d5b5f0",
            "name": "Swimlane Case Connector ",
            "type": ".swimlane",
            "fields": null
        },
        "mappings": [
            {
                "source": "title",
                "target": "caseName",
                "action_type": "overwrite"
            },
            {
                "source": "description",
                "target": "description",
                "action_type": "overwrite"
            },
            {
                "source": "comments",
                "target": "comments",
                "action_type": "append"
            }
        ],
        "version": "WzE2MDc5NzgyLDEyXQ==",
        "error": null,
        "id": "becf0ac0-a635-11ec-9ca2-b9383a2cb913"
    }
]
````

Reply:
Yes, that's the sort of info I saw too @nastasha-solomon (i.e. missing the stuff that appears at the beginning of the existing example):
> {
  "page": 1,
  "per_page": 5,
  "total": 2,
  "cases": [ ...

Reply:
@nastasha-solomon I think you are using the wrong endpoint. The response you should get is

```
[{
  "id":"61787f53-4eee-4741-8df6-8fe84fa616f7",
  "actionTypeId": ".jira",
  "name":"my-Jira",
  "isMissingSecrets":false,
  "config": {
    "apiUrl":"https://elastic.atlassian.net/",
    "projectKey":"ES"
  },
  "isPreconfigured":false,
  "referencedByCount":0
}]
```

I think you are using `/api/cases/configure` instead of `/api/cases/configure/connectors/_find`

Reply:
Ah, yes you are correct. Currently, we only have one Swimlane connector set up and it has a few mappings but nothing else set up. Here's the response I got:

`````
[
    {
        "id": "3331d660-98a6-11ec-8ee2-c385e0d5b5f0",
        "actionTypeId": ".swimlane",
        "name": "Swimlane Case Connector ",
        "isMissingSecrets": false,
        "config": {
            "connectorType": "all",
            "mappings": {
                "alertIdConfig": {
                    "id": "ahrts",
                    "name": "Alert ID",
                    "key": "alert-id",
                    "fieldType": "text"
                },
                "ruleNameConfig": {
                    "id": "anj6n",
                    "name": "Rule Name",
                    "key": "rule-name",
                    "fieldType": "text"
                },
                "severityConfig": {
                    "id": "az40a",
                    "name": "Severity",
                    "key": "severity",
                    "fieldType": "text"
                },
                "caseIdConfig": {
                    "id": "af8h3",
                    "name": "Case ID",
                    "key": "caseid",
                    "fieldType": "text"
                },
                "caseNameConfig": {
                    "id": "at0hz",
                    "name": "Case Name",
                    "key": "case-name",
                    "fieldType": "text"
                },
                "commentsConfig": {
                    "id": "arpja",
                    "name": "Comments",
                    "key": "comments",
                    "fieldType": "comments"
                },
                "descriptionConfig": {
                    "id": "atf44",
                    "name": "Case Description",
                    "key": "case-description",
                    "fieldType": "text"
                }
            },
            "appId": "aM3lUKeaZ46mKrZVy",
            "apiUrl": "https://swimlaneelastic.swimlane.us/"
        },
        "isPreconfigured": false,
        "referencedByCount": 15
    }
]
`````

Reply:
Yes, each connector has a different `config` so it makes sense to have a slightly different response. The format is correct.